tests/test_models_qcnn.py

```diff
@@ -303,3 +305,47 @@ def test_qcnn_training():
     predictions.append(1)
     labels = np.array([[1], [-1], [1]])
     test_qcnn.Accuracy(labels, predictions)
+
+
+def test_two_qubit_ansatz():
```

Comment:
I would pass `backend` as argument of the tests here. With this you will execute the tests in all the backends. Now that the  `qibojit` in-place update is fixed in the main of the `qcnn` model, all the tests should pass.

Reply:
I think it suffices if we test the backends for the training, so we can keep it to the tests that involve training maybe

Reply:
It always better to test the code for all the available backends, in order not to have surprises if an external user decides to use this model with, e.g. `tensorflow` and training it with some automatic differentiation method. 

You only need to add `backend` as an argument of your tests, e.g.: 

```python
def test_two_qubit_ansatz(backend):
     # same code here
```